edpm_ansible/roles/edpm_iscsid/tasks/install.yml

```diff
@@ -14,48 +14,48 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Create persistent directories
+  ansible.builtin.file:
+    path: "{{ item.path }}"
+    setype: "{{ item.setype }}"
+    state: directory
+  loop:
+    - { 'path': /etc/iscsi, 'setype': container_file_t }
+    - { 'path': /etc/target, 'setype': container_file_t }
+    - { 'path': /var/lib/iscsi, 'setype': container_file_t }
 
 - name: Create fcontext entry for iscsi
   community.general.sefcontext:
     target: "{{ item.path }}(/.*)?"
     setype: "{{ item.setype }}"
     state: present
-  with_items:
-    - {'path': /etc/iscsi, 'setype': container_file_t}
-    - {'path': /etc/target, 'setype': container_file_t}
-    - {'path': /var/lib/iscsi, 'setype': container_file_t}
-  when:
-    - edpm_selinux_mode | default('enforcing') == 'enforcing'
-- name: Create persistent directories
-  ansible.builtin.file:
-    path: "{{ item.path }}"
-    state: directory
-    setype: "{{ item.setype }}"
-    mode: 0755
-  with_items:
-    - {'path': /etc/iscsi, 'setype': container_file_t}
-    - {'path': /etc/target, 'setype': container_file_t}
-    - {'path': /var/lib/iscsi, 'setype': container_file_t}
+  loop:
+    - { 'path': /etc/iscsi, 'setype': container_file_t }
+    - { 'path': /etc/target, 'setype': container_file_t }
+    - { 'path': /var/lib/iscsi, 'setype': container_file_t }
+
```

Comment:
Need to add this line https://review.opendev.org/c/openstack/tripleo-ansible/+/875858/2/tripleo_ansible/roles/tripleo_iscsid/tasks/install.yml#36